shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
k - i mean, there is probably something you can do around that. generally i'm against this as the solution because i could see some cases that you might want to have a lazy loaded view with placeholder stuff, then hook in later.

Basically, it seems really weird that the `getHtml` would return an empty string. I think a bit more investigation into the cause of the bug is in-order.